lib/utils.pm

```diff
@@ -1629,4 +1630,24 @@ sub assert_file_content {
     my ($param, $value) = @_;
     assert_script_run("cat $param | grep $value");
 }
+
+=head2 ensure_ca_certificates_suse_installed
+    ensure_ca_certificates_suse_installed();
+
+This functions checks if ca-certificates-suse is installed and if it is not it adds the repository and installs it.
+
+=cut
+
+sub ensure_ca_certificates_suse_installed {
+    if (script_run('rpm -qi ca-certificates-suse') == 1) {
```

Comment:
Should `return unless is_sle;`

Reply:
> very good refactor, it will help consolidate the tests we have for podman and docker.

Thank you @jlausuch! The same refactoring is being done by @punkioudi for `docker.pm` and `podman.pm` test modules.